src/world/Map/TerrainMgr.cpp

```diff
@@ -405,14 +405,13 @@ void TileMap::Load(char* filename)
         return;
     }
 
-    if (header.buildMagic != 12340)  //wow version
+    if (header.buildMagic != sWorld.CacheVersion) // wow version
```

Comment:
can make the macro in MapManagementGlobals.hpp

ADD `#define GAME_BUILD 12340 // wow version`